tests/args.yaml

```diff
@@ -4,7 +4,9 @@ runners:
 
 cases:
  - name: pass args from command line
-   args: hi hey
+   args:
+     - hi
+     - hey
```

Comment:
I did this because I didn't want to just break on spaces (imagine `args: "foo bar" hey`).